Console/retro/Console.cc

```diff
@@ -218,6 +352,8 @@ void Console::PutCharNoUpdate(char c)
         cursorX++;
         if(cursorX >= cols)
             PutCharNoUpdate('\n');
+        // This is to make sure the cursor width is calculated correctly
```

Comment:
Probably another leftover?

Reply:
I agree, the comment is not relevant anymore.

Reply:
Yes, the comment is outdated.